test/integration/eu-west-1/k8s-qa/services/terragrunt.hcl

```diff
@@ -173,6 +173,7 @@ inputs = {
   helm_exporter_github_owner        = "dfds"
   helm_exporter_repo_name           = "platform-manifests-qa"
   helm_exporter_repo_branch         = "main"
+  helm_exporter_target_namespaces   = "kube-system, monitoring"
```

Comment:
I don't think kube-system should be specified. This is a speciel system namespace, so we would never deploy anyting here ourselves.

For QA, I would suggest:

`helm_exporter_target_namespaces = "fluentd, flux-system, monitoring, traefik-green-variant"` 

In the future, when we also have a namespace called traefik-blue-variant, we can add tha too.

Reply:
Good point, by the way have you tested that this works with spaces after the commas?
Perhaps it handles the blank space in the namespace fine.